ce/marketing/solution-uninstall-order.md

```diff
@@ -28,6 +28,42 @@ To delete Dynamics 365 from an instance, first open the Marketing setup wizard a
 
 Marketing solution uninstall order:
 
+Real-time part of marketing:
+1. DynamicsMKT_AnchorSolution
+1. DynamicsMKT_CxpApplicationUser
+1. DynamicsMKT_CIConnection
+1. DynamicsMKT_Roles
+1. msdyn_InsightsAnalyticsMKTAnalytics
+1. msdyn_InsightsAnalyticsMKTConfiguration
+1. msdyn_DataInsightsAndAnalyticsForMKT
+1. DynamicsMKT_Configuration
+1. DynamicsMKT_CustomerVoiceIntegration
+1. DynamicsMKT_StandaloneSitemap
+1. DynamicsMKT_Sitemap
+1. DynamicsMKT_CxpGetStartedExperience
+1. DynamicsMKT_PushNotification
+1. DynamicsMKT_EmailChannel
+1. DynamicsMKT_CxpEmailEditor
+1. DynamicsMKT_CxpEmailEditorCanvas
+1. DynamicsMKT_EmailEditorCanvasShared
+1. DynamicsMKT_EmailEditorShared
+1. DynamicsMKT_PersonalizationUI
+1. DynamicsMKT_CxpCmsIntegration
+1. DynamicsMKT_SmsChannel
+1. DynamicsMKT_OrchestrationAnalytics
+1. DynamicsMKT_CxpOrchestrationUI
+1. DynamicsMKT_Consent
+1. DynamicsMKT_AssetControlsSolution
+1. DynamicsMKT_CxpFormControls
+1. DynamicsMKT_OrchestrationEngineAttachCI
+1. DynamicsMKT_OrchestrationEngine
+1. DynamicsMKT_FeatureConfiguration
+1. DynamicsMKT_BaseSolution
+1. DynamicsMKT_Segmentation
+1. DynamicsMKT_SharedMarketingControls
+1. MicrosoftDynamics_ContentEditor
+
+Outbound part of marketing:
```

Comment:
Not sure what we call the existing Marketing now. "Segment-based marketing" maybe?

Reply:
Or perhaps we can refer to it as "Baseline Marketing application" for the purpose of this document.